packages/firebase_auth/firebase_auth/pubspec.yaml

```
@@ -15,8 +15,8 @@ environment:
 
 dependencies:
   firebase_auth_platform_interface: ^6.10.3
```

```suggestion
  firebase_auth_platform_interface: ^6.10.4
```

It didn't update this 🤔